client/express-checkout/utils/test/normalize.js

```diff
@@ -263,6 +264,66 @@ describe( 'Express checkout normalization', () => {
 		} );
 	} );
 
+	describe( 'normalizePayForOrderData', () => {
+		test( 'should normalize pay for order data with complete event and paymentMethodId', () => {
+			window.wcpayFraudPreventionToken = 'token123';
+
+			const event = {
+				billingDetails: {
+					name: 'John Doe',
+					email: 'john.doe@example.com',
+					address: {
+						organization: 'Some Company',
+						country: 'US',
+						line1: '123 Main St',
+						line2: 'Apt 4B',
+						city: 'New York',
+						state: 'NY',
+						postal_code: '10001',
+					},
+					phone: '(123) 456-7890',
+				},
+				shippingAddress: {
+					name: 'John Doe',
+					organization: 'Some Company',
+					address: {
+						country: 'US',
+						line1: '123 Main St',
+						line2: 'Apt 4B',
+						city: 'New York',
+						state: 'NY',
+						postal_code: '10001',
+					},
+				},
+				shippingRate: { id: 'rate_1' },
+				expressPaymentType: 'express',
+			};
+
+			const paymentMethodId = 'pm_123456';
+
+			expect(
+				normalizePayForOrderData( event, paymentMethodId )
+			).toEqual( {
+				payment_method: 'woocommerce_payments',
+				'wcpay-payment-method': 'pm_123456',
```

Comment:
Nit: This could also be the following:
```javascript
'wcpay-payment-method': paymentMethodId,
```

Reply:
I tend to use literals in one of the sides of the assertion because that's how I avoid being blindsided by the variables. However, since that variable was only being used once, I removed it in [270ff95](https://github.com/Automattic/woocommerce-payments/pull/8994/commits/270ff954e199c5ad1020dc0e95aef03440fad81d)